.github/workflows/pull_request_examples.yml

```diff
@@ -8,35 +8,38 @@ on:
         paths:
             - .github/workflows/pull_request_examples.yml
             - examples/**
+            - packages/**
     push:
         branches:
             - main
             - develop
         paths:
             - .github/workflows/pull_request_examples.yml
             - examples/**
+            - packages/**
     # TODO: Remove and replace with real matrix.
     workflow_dispatch:
         inputs:
             NODE_VERSIONS:
                 description: 'Node versions to test against. Need a JSON array of versions.'
                 required: true
-                default: '[20]'
+                default: '[20, 22]'
                 type: string
 
 permissions:
     contents: read
     pull-requests: read
 
 env:
-    #  NODE_VERSIONS: '[18, 20, 22]'
-    NODE_VERSIONS: '[20]'
+    NODE_VERSIONS: '[20, 22]'
     PATHS: |
         nextjs/starter:
           - .github/workflows/pull_request_examples.yml
           - examples/nextjs/starter/**
+          - packages/**
     GRAPHQL_SCHEMA_FILE: '${{ github.workspace }}/schema.graphql'
     SNAPWP_HELPER_REF: 'develop' # TODO: Get from PR description.
+    TEST_APP_DIR: '${{ runner.temp }}/snapwp-test-app' # Use runner.temp to create app outside the main workspace
```

Comment:
I did not know this was possible. I love it!

@mrrobot47 any concerns we need to be worried about doing it like this?

Reply:
Confirmed via slack : https://rtcamp.slack.com/archives/CK2CEH4F6/p1744969143237679?thread_ts=1744273250.043439&cid=CK2CEH4F6